modules/virt-using-openssh-and-virtctl-port-forward.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-using-openssh-and-virtctl-port-forward.adoc
+
+:_content-type: PROCEDURE
+[id="virt-using-openssh-and-virtctl-port-forward_{context}"]
+= Using OpenSSH and virtctl port-forward
+
+You can use your local OpenSSH client and the `virtctl port-forward` command to connect to a running virtual machine (VM). You can use this method with Ansible to automate the configuration of VMs.
+
+This method is recommended for low-traffic applications because port-forwarding traffic is sent over the control plane. This method is not recommended for high-traffic applications such as Rsync or Remote Desktop Protocol because it places a heavy burden on the API server.

```

Comment:
A single or a low count of RDP sessions will not put a to heavy load on the control plane either. But if multiple VMs suddenly start transferring files with rsync that might be an issue (high traffic).